### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x99 in position 29: invalid start byte

### ISSUE THREAD

**Issue by NikolozCh** [mick88/samp-client#11]
Hello, i wanted to use this library but i fell in this kinda problem. this is code 
```
from samp_client.client import SampClient
record = '46.174.54.192:7777'
ip, port = record.split(":")
with SampClient(address=ip, port=int(port)) as client:
    print (client.get_server_info())
```
It prints out UnicodeDecodeError

also tried add .encode('utf-8').strip() to "client.get_server_info()", but then another error occured

**Comment by mick88** [comment]
I see that the issue is with encoding. The server name contains "™" which is encoded using cp1252 while the library tries to use utf-8 which is incorrect.

**Comment by NikolozCh** [comment]
Wow :) Thanks for replying me after 2+ years lol. I didn't even remember this. We can close the issue

**Comment by mick88** [comment]
Fixed in commit e9df8ec84c397161d706b33043930c8ee57ca87f
